modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/upgrade/v3_7_0/NotificationQueueEntryUpgradeProcess.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.notification.internal.upgrade.v3_7_0;
+
+import com.liferay.notification.constants.NotificationConstants;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Murilo Stodolni
+ */
+public class NotificationQueueEntryUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		runSQL(
+			StringBundler.concat(
+				"update ResourcePermission set name = '",
+				NotificationConstants.RESOURCE_NAME_NOTIFICATION_TEMPLATE,
+				"' where name = 'com.liferay.notification'"));
+
+		runSQL(
+			StringBundler.concat(
+				"update ResourcePermission set primKey = '",
+				NotificationConstants.RESOURCE_NAME_NOTIFICATION_TEMPLATE,
+				"' where primKey = 'com.liferay.notification'"));
+
+		runSQL(
+			"delete from ResourceAction where name = " +
+				"'com.liferay.notification'");
+	}
```

Comment:
@murilostodolni Please check `BasePortletIdUpgradeProcess` -> `updateResourceAction` and `updateResourcePermission`